packages/cli/config/jest.js

```diff
@@ -114,7 +103,7 @@ async function getProjectConfig(targetPath, extraConfig) {
     .join('|');
   if (transformModules.length > 0) {
     console.warn(
-      'The Backstage CLI jest transformModules option is no longer used and will be ignored. All modules are now always transformed.',
+      'The Backstage CLI Jest transformModules option is no longer used and will be ignored. All modules are now always transformed.',
```

Comment:
If you want to break the `jest`->`Jest` change out into a separate PR we could ship that immediately and declutter this one a bit :grin: